### DISCUSSION TITLE
A/B Testing Tool

### DISCUSSION THREAD

**Discussion by demi-johnson** [Bixal/rapid-response-team#96] [Ideas]
Russell Lebo over on Team Fed is working on a project with the [Office of Evaluation Sciences](https://oes.gsa.gov/). They need to create a proof of concept for an A/B testing tool that OES can use in their research on the effectiveness of forms. They want to use it with forms.io. They're working with an outside developer who apparently specializes in this type of thing. 

Maybe RRT can contribute in some sort of way, or use this as inspiration for other tools we can create to support the UX research process.

**Comment by pglevy** [comment]
Do you know what kind of data they're trying to collect? I imaging there are tools that do this, but I'm not sure what questions they're trying to answer, or what metrics they think will help them answer those.
